apps/ws/src/Game.ts

```diff
@@ -71,12 +72,14 @@ export class Game {
         payload: {
           gameId: this.gameId,
           whitePlayer: {
-            name: users.find((user) => user.id === this.player1UserId)?.name,
```

Comment:
Why cant we ues the DB entry? 
Why are we propogating hte name to Game.ts?

Reply:
yes was previous logic to handle guest names when guests where not stored in database, reverted